src/internal/sftpgo/sftpgo.go

```diff
@@ -32,35 +32,29 @@ func rndPass() string {
 	return hex.EncodeToString(data)
 }
 
-type token struct {
-	AccessToken string    `json:"access_token"`
-	ExpiresAt   time.Time `json:"expires_at"`
-}
-
 // API is used to send API requests to the SFTPGo daemon
 type API struct {
 	m       sync.Mutex
 	url     *url.URL
 	login   string
-	pass    string
-	token   *token
+	apikey  string
 	now     func() time.Time
 	do      func(c *http.Client, req *http.Request) ([]byte, error)
 	rndPass func() string
 }
 
 // New returns a new API instance for sending requests to SFTPGo
-func New(apiURL *url.URL, login, pass string) *API {
+// This will need modified for API key use
```

Comment:
I don't understand this comment. Note that for public APIs, the comment style should start with the function name, e.g., `// New does a thing and returns stuff ...`

Reply:
This was just a note to myself when I was initially reviewing the code I'd need to modify that I shouldn't have committed :facepalm: